src/Service.Host/client/src/components/Pallets.js

```diff
@@ -0,0 +1,596 @@
+            storesKittable: pallet.storesKittable,
+            storesKittingPriority: pallet.storesKittingPriority,
+            salesKittable: pallet.salesKittable,
+            salesKittingPriority: pallet.salesKittingPriority,
+            locationTypeId: pallet.locationType == 'LINN' ? 'L' : null,
+            auditedBy: pallet.auditedBy,
+            defaultStockPoolId: pallet.defaultStockPoolId,
+            defaultStockPool: pallet.defaultStockPool,
+            stockType: pallet.stockType,
+            stockState: pallet.stockState,
+            auditFrequencyWeeks: pallet.auditFrequencyWeeks,
+            auditedByDepartmentCode: pallet.auditedByDepartmentCode,
+            mixStates: pallet.mixStates,
+            cage: pallet.cage
+        };
+    }
+
+    return (
+        <Page homeUrl={config.appRoot} showAuthUi={false}>
+            <Grid size={10}>
+                <Typography variant="h4">
+                    {pallet?.palletNumber ? `Pallet ${pallet?.palletNumber}` : 'Pallets'}
+                </Typography>
+            </Grid>
+            {(createPalletLoading ||
+                updateLoading ||
+                isHistoricalEmployeeLoading ||
+                isCurrentEmployeeLoading ||
+                isDepartmentLoading) && (
+                <Grid size={12}>
+                    <Loading />
+                </Grid>
+            )}
+            <Grid container spacing={3}>
+                <Grid size={10}>
+                    {!creating ? (
+                        <Search
+                            autoFocus
+                            propertyName="palletNumber"
+                            label="Pallet Number"
+                            resultsInModal
+                            resultLimit={100}
+                            value={searchPallet}
+                            loading={palletSearchLoading}
+                            handleValueChange={(_, newVal) => setSearchPallet(newVal)}
+                            search={searchPallets}
+                            searchResults={mappedPalletSearchResults}
+                            priorityFunction="closestMatchesFirst"
+                            onResultSelect={handlePalletSearchResultSelect}
+                            clearSearch={clearPallets}
+                        />
+                    ) : (
+                        <InputField
+                            propertyName="palletNumber"
+                            label="Pallet Number"
+                            type="number"
+                            onChange={handleFieldChange}
+                            value={pallet?.palletNumber}
+                            fullWidth
+                        />
+                    )}
+                </Grid>
+
+                <Grid size={2}>
+                    <Button variant="outlined" fullWidth onClick={handleCreatingFieldChange}>
+                        {creating ? `View Pallet` : 'Create Pallet'}
+                    </Button>
```

Comment:
What does this button do? 
 ```
  const handleCreatingFieldChange = () => {
        setPallet({ dateInvalid: null, dateLastAudited: null });
        setCreating(!creating);
    };
    
```
clears out the entirepallet from state which seems odd. certainly unclear behaviour for a button that might say 'View Pallet'